inspectors/osc.py

```diff
@@ -180,7 +180,7 @@ def report_from(result, year, year_range,OUTCOME_CODES):
 
 
 def make_report_id(url):
-  return url.replace('/PublicFiles/','').replace('/publicfiles/','').replace('/','-').replace('.pdf','')
+  return inspector.slugify(url.replace('/PublicFiles/','').replace('/publicfiles/','').replace('.pdf',''))
```

Comment:
Just noting the extra OSC fix in this pull request, unrelated to the Government Attic work. :+1: